contrib/tmux-sessionizer/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "tmux-sessionizer"
+pkgver = "0.4.2"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = []
+makedepends = ["rust-std", "cargo", "pkgconf", "openssl-devel", "libgit2-devel"]
+pkgdesc = (
+    "Tmux Sessionizer: A tool for opening git repositories as tmux sessions"
+)
+maintainer = "Nova <froggo8311@proton.me>"
+license = "MIT"
+url = f"https://github.com/jrmoulton/{pkgname}"
+source = f"{url}/releases/download/v{pkgver}/source.tar.gz"
```

Comment:
```suggestion
source = f"{url}/releases/download/v{pkgver}/source.tar.gz>source-{pkgver}.tar.gz"
```

this has to be renamed to not conflict

alternatively use /archive/refs/tags/v{pkgver}.tar.gz and not the 'source tarball' since it probably does not have anything needed in it

Reply:
by alternatively i mean you should use `f"{url}/archive/refs/tags/v{pkgver}.tar.gz"` unless the source.tar.gz is required for some reason

Reply:
it shouldn't be required, let me test to see if your way works

Reply:
ok done